### PR TITLE
Disabled restore button for non-off VMs

### PR DESCRIPTION
**Pull request by bond95**
Fixes: https://github.com/oVirt/ovirt-web-ui/issues/979

### CODE REVIEW

File: src/components/VmDetails/cards/SnapshotsCard/index.js [hunk 1]
```diff
@@ -75,7 +80,7 @@ const SnapshotsCard = ({ vm }) => {
       idPrefix={idPrefix}
       editable={false}
     >
-      <ConnectedSnapshots snapshots={snapshots} vmId={vm.get('id')} canUserManipulateSnapshot={vm.get('canUserManipulateSnapshots')} idPrefix={idPrefix} />
+      <ConnectedSnapshots snapshots={snapshots} vmId={vm.get('id')} canUserManipulateSnapshot={vm.get('canUserManipulateSnapshots')} idPrefix={idPrefix} isVmDown={vm.get('status') === DOWN_STATUS} />
```
Comment (sjd78) [inline src/components/VmDetails/cards/SnapshotsCard/index.js]: Nit-pick: This line is too long.  Splitting it to have each prop to its own line would be easier to read and have consistent styling with the rest of the codebase.

**Comment by bond95** [comment]
@leistnerova WDYT?

**Comment by leistnerova** [comment]
Icon next to snapshot description is disabled, but restore button in info not.
Screenshot from 2019-05-06 10-43-28

**Comment by bond95** [comment]
@leistnerova 
image

**Comment by leistnerova** [comment]
LGTM

**Pull request merged**